avocado/utils/distro.py

```diff
@@ -373,6 +373,18 @@ def get_distro(self):
         return distro
 
 
+class OpenEulerProbe(Probe):
+
+    """
+    Simple probe for openEuler systems in general
+    """
+
+    CHECK_FILE = '/etc/openEuler-release'
+    CHECK_FILE_CONTAINS = 'openEuler release'
+    CHECK_FILE_DISTRO_NAME = 'openEuler'
+    CHECK_VERSION_REGEX = re.compile(r'openEuler release (\d+)\.(\d+).*')
```

Comment:
Hi @wang-shengfang, do you think `-LTS` should be part of the version or not? See #4073 about that (it currently considers it part of the version "number").

Reply:
Hi @clebergnu ，LTS is a codename，I don't think it's necessary, and VERSION_ID in /etc/os-release does not contain LTS.

/etc/openEuler-release：
<pre>openEuler release 20.03 (LTS)</pre>
/etc/os-release：
<pre>NAME="openEuler"
VERSION="20.03 (LTS)"
ID="openEuler"
VERSION_ID="20.03"
PRETTY_NAME="openEuler 20.03 (LTS)"
ANSI_COLOR="0;31"</pre>